package.json

```diff
@@ -110,6 +110,7 @@
     ]
   },
   "dependencies": {
+    "@aws-crypto/client-node": "^3.1.1",
```

Comment:
could you include the `yarn.lock` change? I assume since we added new dependency, yarn.lock is supposed to be updated

Reply:
> Yarn resolves these issues around versioning and non-determinism by using lock files and an install algorithm that is deterministic and reliable. These lock files lock the installed dependencies to a specific version and ensure that every install results in the exact same file structure in node_modules across all machines.

https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

Reply:
sure, didn't know this is a convention here.

Reply:
nvm, mani already added yarn lock in one PR